default/python/AI/InvasionAI.py

```diff
@@ -43,7 +43,7 @@ def get_invasion_fleets():
                                  (sys_id != INVALID_ID) and universe.systemsConnected(sys_id, home_system_id,
                                                                                       empire_id)]
     else:
-        print "Warning: Empire has no identifiable homeworld; will treat all visible planets as accessible."
+        warn("Empire has no identifiable homeworld; will treat all visible planets as accessible.")
```

Comment:
It seems not erroneous that an empire could still have ships but no remaining homeworld, so this should be just debug not warn, I think.

Reply:
I am a simple man, I see `Warning` I use `warn`.